csi/nodeserver.py

```diff
@@ -107,3 +107,38 @@ def NodeGetInfo(self, request, context):
         return csi_pb2.NodeGetInfoResponse(
             node_id=os.environ["NODE_ID"],
         )
+
+    def NodeExpandVolume(self, request, context):
+        start_time = time.time()
+        hostvol = request.volume_context.get("hostvol", "")
+        pvpath = request.volume_context.get("path", "")
+        pvtype = request.volume_context.get("pvtype", "")
+        voltype = request.volume_context.get("type", "")
+        #gserver = request.volume_context.get("gserver", None)
+        #gvolname = request.volume_context.get("gvolname", None)
+        #options = request.volume_context.get("options", None)
+
+        #mntdir = os.path.join(HOSTVOL_MOUNTDIR, hostvol)
+
+        #pvpath_full = os.path.join(mntdir, pvpath)
+
+        logging.debug(logf(
+            "Received the expand request",
+            volume=request.volume_id,
+            voltype=voltype,
+            hostvol=hostvol,
+            pvpath=pvpath,
+            pvtype=pvtype
+        ))
+
+        expand_volume(request.target_path)
```

Comment:
Is it called for RWX volume? if yes, then we don't need to do anything at all, as quota update is good enough. Only for Block image, we need to do `grow_xfs`.